DEPS

```diff
@@ -525,6 +525,18 @@ deps = {
     'dep_type': 'cipd',
   },
 
+  # TODO(fxb/4443): Remove this when Fuchsia can provide the Windows Clang Toolchain
+  'src/third_party/llvm-build/Release+Asserts': {
```

Comment:
Can't this just be `//build/toolchain` like for all other platforms?

Reply:
Right now the buildroot is looking for it here.  We can patch buildroot for this, but I think for now this makes it clearer we're not using Fuchsia clang.  WDYT?